juliac.jl

```diff
@@ -63,12 +67,13 @@ function main(args)
         parsed_args["shared"],
         parsed_args["executable"],
         parsed_args["julialibs"],
-        parsed_args["clean"]
+        parsed_args["clean"],
+        parsed_args["target"]
     )
 end
 
 function julia_compile(julia_program, c_program=nothing, build_dir="builddir", verbose=false, quiet=false,
-                       object=false, shared=false, executable=true, julialibs=true, clean=false)
+                       object=false, shared=false, executable=true, julialibs=true, clean=false, target="x86-64")
```

Comment:
default to "generic"

Reply:
@vtjnash, I think that when `juliac.jl` is run without `--cpu-target` option, the default should be to run `julia` without `--cpu-target` option as well, this way `juliac.jl` would have same default as `julia`, do you agree?

Reply:
Yes, I suppose that makes sense. Maybe make the default the empty string then, and don't override the cpu target in that case?

Reply:
Yes that's what I meant. PR is coming.